app/repository/event.py

```diff
@@ -87,7 +87,7 @@ def all(db: Session) -> list[EventReadDTO]:
     :param db Session: The database connection.
     :return Optional[EventReadDTO]: All family events in the database.
     """
-    family_event_metas = _get_query(db).all()
+    family_event_metas = _get_query(db).order_by(desc(FamilyEvent.last_modified)).all()
```

Comment:
I'm wondering if we remove this from events ... as looking at the sqlalchemy the events on a family are ordered by the event's datetime... which seems to make more sense than a modified date/time but gonna approve anyway as its somthing we can address later